src/main/java/Duke.java

```diff
@@ -1,10 +1,42 @@
+import duke.commands.Command;
```

Comment:
I like that you packaged your files rather neatly and your code is neat too! It's easy to follow along